src/sass/_variables.scss

```diff
@@ -6,6 +6,9 @@ $footer-bg: #363636;
 
 $primary: #d58e32;
```

Comment:
i'd change it to `$primary-color` to be consistent

Reply:
@xkxd `.category:hover:hover` my mistake I missed one bracket :) also selector created in `ul il` list you can double check it.
I would rather not change this variable, because this variable was created by mentors on creating . And if I change it i will have to change everywhere ( I mean in every file where it was used)

Reply:
If you don't like it, one of you may create a task for this "code issue" and give it a low priority and 1 point.